apps/meteor/server/startup/migrations/v298.ts

```diff
@@ -0,0 +1,25 @@
+import { settings, settingsRegistry } from '../../../app/settings/server';
+import { addMigration } from '../../lib/migrations';
+
+addMigration({
+	version: 293,
+	async up() {
+		const customOauthServices = settings.getByRegexp(/Accounts_OAuth_Custom-[^-]+$/im);
```

Comment:
please a model and do a query instead.. it is not safe to use the settings cache during migrations